src/main/scala/io/iohk/praos/domain/BlockchainManager.scala

```diff
@@ -0,0 +1,42 @@
+package io.iohk.praos.domain
+
+case class BlockchainManager() {
```

Comment:
Maybe we could rename this class to something like `ConsensusResolver` or sth similar?

Reply:
totally agree!